syntax = "proto3";

package simple;

// The hello service definition.
service HelloService {
    // Sends a hello
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
    string name = 1;
}

// The response message containing the greetings.
message HelloReply {
    string message = 1;
}
